nuntium/templates/nuntium/profiles/welcome.html

```diff
@@ -4,9 +4,107 @@
 {% block content %}
 
 <div class="page-header">
-    <h2>{% trans "Welcome" %}</h2>
+    <h2>
+       {% blocktrans with name=writeitinstance.name %}
+         Welcome! This is {{ name }}
+       {% endblocktrans %}
+    </h2>
 </div>
 
-<p>{% trans "Write to your representative etc." %}</p>
+<p>
+  {% blocktrans with name=writeitinstance.name %}
+    This is your &ldquo;{{name}}&rdquo; message-sending site.
+  {% endblocktrans %}
+</p>
+{% if writeitinstance.config.testing_mode %}
+  <p>
+    {% blocktrans %}
+      You're currently running in <strong>testing mode</strong>, so all
+      the emails sent will go to you (and not the recipients).
+      Contact us if you want this restriction lifted and you're ready to
+      send real messages to real people!
+    {% endblocktrans %}
+  </p>
+{% endif %}
+<p>
+  {% if writeitinstance.persons.count %}
```

Comment:
Ideally here we'd want to also check that there are *contactable* contacts, rather than just contacts (e.g. with no email addresses, or who aren't marked as Active)